docs/docs/ref-03-component-specs.md

```diff
@@ -37,6 +37,10 @@ object getInitialState()
 
 Invoked once before the component is mounted. The return value will be used as the initial value of `this.state`.
 
+> Note:
+>
+> This method is not available on ES6 `class` components that extend `React.Component`. It may be removed entirely in a future version of React.
```

Comment:
Would it be useful to link to the ES6 Classes docs in these notes? Without more description, someone might be lost when they find this and wonder how they should handle initial state if not with `getInitialState`.

ES6 Classes: https://facebook.github.io/react/docs/reusable-components.html#es6-classes